### PR TITLE
Browser Router V6

### PR DESCRIPTION
**Pull request by Lyanthropos**
Apache config needs testing before merging info prod.

### CODE REVIEW

File: src/Api/getAllAssignmentSettings.php [hunk 1]
```diff
@@ -74,16 +74,15 @@
           "proctorMakesAvailable" => $row['proctorMakesAvailable'] == '1' ? true : false,
           "doenetId" => $row['doenetId']
     );
+  } else {
```
Comment (Lyanthropos) [inline src/Api/getAllAssignmentSettings.php:77]: @djnykamp is this what you wanted to be removed? Not sure why its back but want to review with you

**Review by Lyanthropos — COMMENTED** [review]
One spot to review with Duane. Then ready for a test deployment on dev.

File: src/Api/getAllAssignmentSettings.php [hunk 2]
```diff
@@ -74,16 +74,15 @@
...
+    http_response_code(404);
```
Comment (Lyanthropos) [inline src/Api/getAllAssignmentSettings.php:78]: @djnykamp is this what you wanted to be removed? Not sure why its back but want to review it with you

**Comment by Lyanthropos** [comment]
@kevincharles Ready for deployment on dev

**Comment by Lyanthropos** [comment]
We do have an update for the apache conf

**Review by clary149 — APPROVED** [review]

**Pull request merged**